libs/blocks/faas-config/faas-config.js

```diff
@@ -425,8 +425,8 @@ const Configurator = ({ rootEl }) => {
 
   useEffect(() => {
     if (isFaasLoaded) {
-      initFaas(state, document.getElementsByClassName('faas')[0]);
       saveStateToLocalStorage(state);
+      initFaas(JSON.parse(localStorage.getItem(LS_KEY)), document.getElementsByClassName('faas')[0]);
```

Comment:
Why the change to reading from localstorage?  Isn't `state` the exact same?

Reply:
@chrischrischris This PR is trying to set state.as as false if it is configuration page.
By this change, the faas config renders form as `state.as` false even though user sets as true.